### PR TITLE
doc: add minutes for meeting 16 June 2020

### PR DESCRIPTION
**Pull request by mhdawson**
Signed-off-by: Michael Dawson <michael_dawson@ca.ibm.com>

### CODE REVIEW

File: meetings/2020-06-16.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+﻿# Node.js  Package Maintenance Team Meeting 2020-06-16
+## Links
+
+* **Recording**: https://youtu.be/D47AoyjM2cM
```
Comment (mhdawson) [inline meetings/2020-06-16.md:4]:
```
https://github.com/nodejs/package-maintenance/issues/371
```

File: meetings/2020-06-16.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+﻿# Node.js  Package Maintenance Team Meeting 2020-06-16
+## Links
+
+https://github.com/nodejs/package-maintenance/issues/371
```
Comment (mhdawson) [inline meetings/2020-06-16.md:4]:
```suggestion
* **Recording**: https://github.com/nodejs/package-maintenance/issues/371
```

**Comment by larson-carter** [comment]
@mhdawson I haven't been receiving any notifications? Is there a github team that I am supposed to be added to?

**Comment by mhdawson** [comment]
@larson-carter I've added you to the team now, but I don't think that results in notifications. The meetings are scheduled on the calendar here nodejs.org/calendar and and issue is created a few days in advance which I think all you would need to do in order to get notifications is to follow the repository.

**Comment by larson-carter** [comment]
@mhdawson Thank you, after joining the github team, it added me to the proper watched repos. I also setup the calendar to sync with all of my google accounts. So now I should be getting notifications about every repo update and when every meeting is.

Thank You!

**Pull request merged**